attestation-agent/lib/src/config/mod.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2024 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::{anyhow, Result};
+use serde::Deserialize;
+use std::fs::File;
+
+pub mod aa_kbc_params;
+
+pub const DEFAULT_AA_CONFIG_PATH: &str = "/etc/attestation-agent.toml";
+
+#[derive(Clone, Debug, Deserialize)]
+#[allow(dead_code)]
+pub struct Config {
+    /// URL Address of Attestation Service
+    pub as_url: String,
+    // TODO: Add more fields that accessing AS needs.
+}
+
+impl TryFrom<&str> for Config {
+    type Error = anyhow::Error;
```

Comment:
In (new) library code, we should consider using concrete error types (`thiserror` crate makes this easy) in library code. this would probably translate to something like this: 

```rust
use thiserror::Error;

#[derive(Error, Debug)]
pub enum ConfigFileError {
    #[error("failed to open")]
    Io(#[from] io::Error), 
    #[error("failed to parse")]
    Parse(#[from] serde_json::Error),     
}

impl TryFrom<&str> for Confg {
    type Error = ConfigFileError;
    fn try_from(config_path: &str) -> Result<Self, Self::Error> {
        let file = File::open(config_path)?;
        let cfg: Config = serde_json::from_reader(file)?;
        Ok(cfg)
    }
}
```